az-core/src/test/java/azkaban/utils/ExecutorServiceUtilsTest.java

```diff
@@ -57,7 +57,8 @@ public void force_shutdown_after_timeout() throws InterruptedException {
     assertThat(service.isShutdown()).isTrue();
     final long shutdownDuration = endShutdownTime - beginShutdownTime;
     // Give some buffer for overhead to reduce false positives.
-    assertThat(shutdownDuration).isLessThan(10);
+    // 100 is still much smaller than 1000 if the task were not forcefully terminated.
+    assertThat(shutdownDuration).isLessThan(100);
```

Comment:
That's milliseconds? 100 would've still been way too low as sleep time to stabilize some flaky unit tests before as Travis builds seem to suffer from random slowdowns. Even 1 second wasn't enough sometimes. If it was up to me, I'd remove this assertion if it can't be set to something that's guaranteed to be high enough, like 10 seconds.

Reply:
Yes, that's milliseconds. 

I'd like to see how often this operation takes longer than 100ms. The worst I have seen so far took 11ms.

Reply:
Maybe this operation is then more immune to random slowdowns.. I wonder if Travis can randomly have some longer GC pauses or such that cause unexpected extra delay between operations. But this is just speculation, I didn't ever measure the times on Travis and see how much they vary.

Reply:
@juhoautio 
I thought more about your suggestion. I think you are right. I increased the threshold.

@kunkun-tang 
The time is how long it took to interrupt the thread that is sleeping. The 11ms was seen on travis.